apps/src/templates/teacherHomepage/ManageSectionsCollapsible.story.jsx

```diff
@@ -39,6 +39,7 @@ export default storybook => {
                 sectionCode: "HPRWHG"
               },
             ]}
+            codeOrgUrlPrefix = "http://localhost:3000/"
```

Comment:
Agree that storybook isn't the best place to test links. Also, I'm realizing that since this is the code.org url, my suggestion of using "localhost-studio.code.org" is entirely wrong.

Given that the link itself doesnt really matter, I wonder if using "http://code.org/" here will be clearer for indicating what the expected value is.